membership-attribute-service/app/components/TouchpointComponents.scala

```diff
@@ -143,20 +143,20 @@ class TouchpointComponents(
   )
 
   lazy val futureCatalog: Future[CatalogMap] = catalogService.catalog
-    .map(_.fold[CatalogMap](error => { println(s"error: ${error.list.toList.mkString}"); Map() }, _.map))
+    .map(_.fold[CatalogMap](error => { logger.errorNoPrefix(scrub"error: ${error.list.toList.mkString}"); Map() }, _.map))
     .recover { case error =>
-      logger.error(scrub"Failed to load the product catalog from Zuora due to: $error")
+      logger.errorNoPrefix(scrub"Failed to load the product catalog from Zuora due to: $error")
```

Comment:
if there's no relevant identity id I made a `...NoPrefix` method.

Another option (used by play framework) would be to provide a low priority implicit value of LogPrefix, however the drawback of that is if we forget to take the identity id as a parameter it would still compile but wouldn't log the identity id.  I'd rather have a compiler error unless we explicitly say it's NoPrefix.